src/main/java/frc/robot/subsystems/pivot/PivotIOTalonFX.java

```diff
@@ -231,7 +246,12 @@ public void setPivotFromSpeakerDistance(double speakerDistance) {
     setPivotAngle(speakerAngle);
   }
 
-  /** Sets Pivot based on Pass Distance */
+  /**
+   * Uses distance in meters from the passing position to set the pivot angle (degrees) of the
```

Comment:
what is the passing position? did we write code for passing?! sick

Reply:
what

Reply:
like passing game pieces

Reply:
my guy

Reply:
this is the same code as last hear

Reply:
thats the entire point of the pr